components/html-block/README.md

```diff
@@ -99,29 +100,41 @@ Examples are provided to display how user-authored math can be embedded within y
 </d2l-html-block>
 ```
 
-**LaTeX:** Rendering LaTeX requires the `us125413-mathjax-render-latex` feature flag to be enabled.
+**LaTeX:** Rendering LaTeX requires the `mathjax` context to be set correctly. For testing and or demo pages you can do the following.
 
 <!-- docs: demo code -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/html-block/html-block.js';
-
-  window.D2L = {};
-  D2L.LP = {};
-  D2L.LP.Web = {};
-  D2L.LP.Web.UI = {};
-  D2L.LP.Web.UI.Flags = {
-    Flag: (feature, defaultValue) => {
-      if (feature === 'us125413-mathjax-render-latex') return true;
```

Comment:
The `us125413-mathjax-render-latex` flag isn't used anywhere and isn't required for LaTeX to work from what I can tell.

Reply:
Was marked as removed as part of https://github.com/Brightspace/lms-launch-darkly-flags/pull/2925 and later deleted